util/mbed/mbed_cmake_vargen.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
```

Comment:
This is what generates that cmake mbed variables file. Not called from anywhere currently, but should be ideally. Not really a high priority though.